### ISSUE TITLE
Updated environment needs fixes

### ISSUE THREAD

**Issue by ekatef** [ekatef/heat-peaks-lit#2]
The environment update with #1 leads to some weird errors when running the app like `AttributeError: '_config' object has no attribute 'initialized'`. It seems to be an environment problem, as [here](https://stackoverflow.com/questions/76506068/how-to-solve-attributeerror-config-object-has-no-attribute-variables-for-p) or [here](https://discuss.streamlit.io/t/getting-attributeerror-module-streamlit-has-no-attribute-set-page-config/16507) and can be resolved by using the old environment.

**Comment by ekatef** [comment]
A working (albeit outdated) environment looks like follows:
```bash
# Name                    Version                   Build  Channel
altair                    4.2.2                    pypi_0    pypi
anyio                     3.7.1              pyhd8ed1ab_0    conda-forge
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0          py310h90acd4f_3    conda-forge
arrow                     1.2.3              pyhd8ed1ab_0    conda-forge
asttokens                 2.2.1              pyhd8ed1ab_0    conda-forge
attrs                     23.1.0             pyh71513ae_1    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                pyhd8ed1ab_3    conda-forge
backports.functools_lru_cache 1.6.5              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.12.2             pyha770c72_0    conda-forge
bleach                    6.0.0              pyhd8ed1ab_0    conda-forge
blinker                   1.6.2                    pypi_0    pypi
blosc                     1.21.4               heccf04b_0    conda-forge
bokeh                     2.4.3              pyhd8ed1ab_3    conda-forge
boost-cpp                 1.78.0               hf5ba120_3    conda-forge
bottleneck                1.3.7           py310h936d966_0    conda-forge
branca                    0.6.0              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                hb7f2c08_9    conda-forge
brotli-bin                1.0.9                hb7f2c08_9    conda-forge
brotli-python             1.0.9           py310h7a76584_9    conda-forge
bzip2                     1.0.8                h0d85af4_4    conda-forge
c-ares                    1.19.1               h0dc2134_0    conda-forge
c-blosc2                  2.10.0               h0dcb076_0    conda-forge
ca-certificates           2023.7.22            h8857fd0_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
cachetools                5.3.1                    pypi_0    pypi
cairo                     1.16.0            h09dd18c_1016    conda-forge
cartopy                   0.21.1          py310h8b10423_1    conda-forge
certifi                   2023.7.22          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1          py310ha78151a_3    conda-forge
cfitsio                   4.2.0                hd56cc12_0    conda-forge
cftime                    1.6.2           py310h936d966_1    conda-forge
charset-normalizer        3.2.0              pyhd8ed1ab_0    conda-forge
click                     8.1.6           unix_pyh707e725_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cligj                     0.7.2              pyhd8ed1ab_1    conda-forge
cloudpickle               2.2.1              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
colorcet                  3.0.1                      py_0    pyviz
comm                      0.1.3              pyhd8ed1ab_0    conda-forge
curl                      8.2.1                h5f667d7_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
dask-core                 2023.7.1           pyhd8ed1ab_0    conda-forge
datashader                0.15.1                     py_0    pyviz
datashape                 0.5.4                      py_1    conda-forge
debugpy                   1.6.7           py310h7a76584_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
deprecation               2.1.0              pyh9f0ad1d_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
exceptiongroup            1.1.2              pyhd8ed1ab_0    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
expat                     2.5.0                hf0c8a7f_1    conda-forge
fiona                     1.9.4           py310h10f461c_0    conda-forge
flit-core                 3.9.0              pyhd8ed1ab_0    conda-forge
folium                    0.14.0             pyhd8ed1ab_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.14.2               h5bb23bf_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.41.1          py310h6729b98_0    conda-forge
fqdn                      1.5.1              pyhd8ed1ab_0    conda-forge
freetype                  2.12.1               h3f81eb7_1    conda-forge
freexl                    1.0.6                hb7f2c08_1    conda-forge
fsspec                    2023.6.0           pyh1a96a4e_0    conda-forge
gdal                      3.7.0           py310h496d566_1    conda-forge
geopandas                 0.13.0             pyhd8ed1ab_0    conda-forge
geopandas-base            0.13.0             pyha770c72_0    conda-forge
geos                      3.11.2               hf0c8a7f_0    conda-forge
geotiff                   1.7.1                h75a88c4_8    conda-forge
geoviews                  1.9.6                      py_0    pyviz
geoviews-core             1.9.6                      py_0    pyviz
gettext                   0.21.1               h8a4c099_0    conda-forge
giflib                    5.2.1                hb7f2c08_3    conda-forge
gitdb                     4.0.10                   pypi_0    pypi
gitpython                 3.1.32                   pypi_0    pypi
glib                      2.76.4               h7d26f99_0    conda-forge
glib-tools                2.76.4               h7d26f99_0    conda-forge
gst-plugins-base          1.22.5               hb5d3a86_0    conda-forge
gstreamer                 1.22.5               h840fbdc_0    conda-forge
hdf4                      4.2.15               h9804679_6    conda-forge
hdf5                      1.14.0          nompi_hbf0aa07_103    conda-forge
holoviews                 1.16.0                     py_0    pyviz
hvplot                    0.8.3                      py_0    pyviz
icu                       72.1                 h7336db1_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.8.0              pyha770c72_0    conda-forge
importlib_metadata        6.8.0                hd8ed1ab_0    conda-forge
importlib_resources       6.0.0              pyhd8ed1ab_1    conda-forge
ipykernel                 6.25.0             pyh5fb750a_0    conda-forge
ipython                   8.14.0             pyhd1c38e8_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                8.0.7              pyhd8ed1ab_0    conda-forge
isoduration               20.11.0            pyhd8ed1ab_0    conda-forge
jedi                      0.18.2             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
joblib                    1.3.0              pyhd8ed1ab_1    conda-forge
json-c                    0.16                 h01d06f9_0    conda-forge
jsonpointer               2.0                        py_0    conda-forge
jsonschema                4.18.4             pyhd8ed1ab_0    conda-forge
jsonschema-specifications 2023.7.1           pyhd8ed1ab_0    conda-forge
jsonschema-with-format-nongpl 4.18.4             pyhd8ed1ab_0    conda-forge
jupyter                   1.0.0           py310h2ec42d9_8    conda-forge
jupyter_client            8.3.0              pyhd8ed1ab_0    conda-forge
jupyter_console           6.6.3              pyhd8ed1ab_0    conda-forge
jupyter_core              5.3.1           py310h2ec42d9_0    conda-forge
jupyter_events            0.6.3              pyhd8ed1ab_1    conda-forge
jupyter_server            2.7.0              pyhd8ed1ab_0    conda-forge
jupyter_server_terminals  0.4.4              pyhd8ed1ab_1    conda-forge
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        3.0.8              pyhd8ed1ab_0    conda-forge
kealib                    1.5.1                hde10d81_3    conda-forge
kiwisolver                1.4.4           py310ha23aa8a_1    conda-forge
krb5                      1.21.1               hb884880_0    conda-forge
lcms2                     2.15                 h2dcdeff_1    conda-forge
lerc                      4.0.0                hb486fe8_0    conda-forge
libaec                    1.0.6                hf0c8a7f_1    conda-forge
libarchive                3.6.2                h0b5dc4a_1    conda-forge
libblas                   3.9.0           17_osx64_openblas    conda-forge
libbrotlicommon           1.0.9                hb7f2c08_9    conda-forge
libbrotlidec              1.0.9                hb7f2c08_9    conda-forge
libbrotlienc              1.0.9                hb7f2c08_9    conda-forge
libcblas                  3.9.0           17_osx64_openblas    conda-forge
libclang                  15.0.7          default_hdb78580_3    conda-forge
libclang13                15.0.7          default_h953c2e9_3    conda-forge
libcurl                   8.2.1                h5f667d7_0    conda-forge
libcxx                    16.0.6               hd57cbcb_0    conda-forge
libdeflate                1.18                 hac1461d_0    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libexpat                  2.5.0                hf0c8a7f_1    conda-forge
libffi                    3.4.2                h0d85af4_5    conda-forge
libgdal                   3.7.0                hd7102d7_1    conda-forge
libgfortran               5.0.0           11_3_0_h97931a8_32    conda-forge
libgfortran5              12.2.0              he409387_32    conda-forge
libglib                   2.76.4               hc62aa5d_0    conda-forge
libiconv                  1.17                 hac89ed1_0    conda-forge
libjpeg-turbo             2.1.5.1              hb7f2c08_0    conda-forge
libkml                    1.3.0             haeb80ef_1015    conda-forge
liblapack                 3.9.0           17_osx64_openblas    conda-forge
libllvm14                 14.0.6               hc8e404f_4    conda-forge
libllvm15                 15.0.7               he4b1e75_3    conda-forge
libnetcdf                 4.9.2           nompi_hf077d52_105    conda-forge
libnghttp2                1.52.0               he2ab024_0    conda-forge
libogg                    1.3.4                h35c211d_1    conda-forge
libopenblas               0.3.23          openmp_h429af6e_0    conda-forge
libopus                   1.3.1                hc929b4f_1    conda-forge
libpng                    1.6.39               ha978bb4_0    conda-forge
libpq                     15.3                 h3df487d_2    conda-forge
librttopo                 1.1.0               h5c328d2_13    conda-forge
libsodium                 1.0.18               hbcb3906_1    conda-forge
libspatialindex           1.9.3                he49afe7_4    conda-forge
libspatialite             5.0.1               h8e1627f_26    conda-forge
libsqlite                 3.42.0               h58db7d2_0    conda-forge
libssh2                   1.11.0               hd019ec5_0    conda-forge
libtiff                   4.5.1                hf955e92_0    conda-forge
libvorbis                 1.3.7                h046ec9c_0    conda-forge
libwebp-base              1.3.1                h0dc2134_0    conda-forge
libxcb                    1.15                 hb7f2c08_0    conda-forge
libxml2                   2.11.4               hd95e348_0    conda-forge
libzip                    1.9.2                h6db710c_1    conda-forge
libzlib                   1.2.13               h8a1eda9_5    conda-forge
linopy                    0.2.3              pyhd8ed1ab_0    conda-forge
llvm-openmp               16.0.6               hff08bdf_0    conda-forge
llvmlite                  0.40.1          py310hd8379ad_0    conda-forge
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
lz4-c                     1.9.4                hf0c8a7f_0    conda-forge
lzo                       2.10              haf1e3a3_1000    conda-forge
mapclassify               2.5.0              pyhd8ed1ab_1    conda-forge
markdown                  3.4.4              pyhd8ed1ab_0    conda-forge
markdown-it-py            3.0.0                    pypi_0    pypi
markupsafe                2.1.3           py310h6729b98_0    conda-forge
matplotlib                3.5.2           py310h2ec42d9_1    conda-forge
matplotlib-base           3.5.2           py310h4510841_1    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mdurl                     0.1.2                    pypi_0    pypi
mistune                   3.0.0              pyhd8ed1ab_0    conda-forge
multipledispatch          0.6.0                      py_0    conda-forge
munch                     4.0.0              pyhd8ed1ab_0    conda-forge
munkres                   1.0.7                      py_1    bioconda
mysql-common              8.0.33               hc6116ba_2    conda-forge
mysql-libs                8.0.33               haa61052_2    conda-forge
nbclassic                 1.0.0              pyhb4ecaf3_1    conda-forge
nbclient                  0.8.0              pyhd8ed1ab_0    conda-forge
nbconvert                 7.7.3              pyhd8ed1ab_0    conda-forge
nbconvert-core            7.7.3              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.7.3              pyhd8ed1ab_0    conda-forge
nbformat                  5.9.1              pyhd8ed1ab_0    conda-forge
ncurses                   6.4                  hf0c8a7f_0    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
netcdf4                   1.6.3           nompi_py310h344eb58_102    conda-forge
networkx                  3.1                pyhd8ed1ab_0    conda-forge
notebook                  6.5.4              pyha770c72_0    conda-forge
notebook-shim             0.2.3              pyhd8ed1ab_0    conda-forge
nspr                      4.35                 hea0b92c_0    conda-forge
nss                       3.89                 h78b00b3_0    conda-forge
numba                     0.57.1          py310he09a53b_0    conda-forge
numexpr                   2.8.4           py310h5e4fcda_1    conda-forge
numpy                     1.23.5                   pypi_0    pypi
openjpeg                  2.5.0                h13ac156_2    conda-forge
openssl                   3.1.1                h8a1eda9_1    conda-forge
overrides                 7.3.1              pyhd8ed1ab_0    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    2.0.1           py310h5e4fcda_1    conda-forge
pandoc                    3.1.3                h9d075a6_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
panel                     0.14.4                     py_0    pyviz
param                     1.13.0                     py_0    pyviz
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
partd                     1.4.0              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                h1c4e4bc_0    conda-forge
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    10.0.0          py310hd63a8c7_0    conda-forge
pip                       23.1.2             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               hbcb3906_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_0    conda-forge
platformdirs              3.9.1              pyhd8ed1ab_0    conda-forge
plotly                    5.14.1             pyhd8ed1ab_0    conda-forge
ply                       3.11                       py_1    conda-forge
pooch                     1.7.0              pyha770c72_3    conda-forge
poppler                   23.05.0              he041c3a_1    conda-forge
poppler-data              0.4.12               hd8ed1ab_0    conda-forge
postgresql                15.3                 hc940a54_2    conda-forge
proj                      9.2.0                hf909084_0    conda-forge
prometheus_client         0.17.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.39             pyha770c72_0    conda-forge
prompt_toolkit            3.0.39               hd8ed1ab_0    conda-forge
protobuf                  3.20.3                   pypi_0    pypi
psutil                    5.9.5           py310h90acd4f_0    conda-forge
pthread-stubs             0.4               hc929b4f_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py-cpuinfo                9.0.0              pyhd8ed1ab_0    conda-forge
pyarrow                   12.0.1                   pypi_0    pypi
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pyct                      0.5.0                      py_0    pyviz
pyct-core                 0.5.0                      py_0    pyviz
pydeck                    0.8.1b0                  pypi_0    pypi
pygments                  2.15.1             pyhd8ed1ab_0    conda-forge
pympler                   1.0.1                    pypi_0    pypi
pyobjc-core               9.2             py310hef2d279_0    conda-forge
pyobjc-framework-cocoa    9.2             py310hef2d279_0    conda-forge
pyomo                     6.6.1           py310h9e9d8ca_0    conda-forge
pyparsing                 3.1.0              pyhd8ed1ab_0    conda-forge
pyproj                    3.5.0           py310h7504ac3_1    conda-forge
pypsa                     0.23.0             pyhd8ed1ab_0    conda-forge
pyqt                      5.15.9          py310hecc045f_4    conda-forge
pyqt5-sip                 12.12.2         py310h018f80b_4    conda-forge
pyshp                     2.3.1              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
pytables                  3.8.0           py310he6142c8_1    conda-forge
python                    3.10.12         had23ca6_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.18.0             pyhd8ed1ab_0    conda-forge
python-json-logger        2.0.7              pyhd8ed1ab_0    conda-forge
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python_abi                3.10                    3_cp310    conda-forge
pytz                      2023.3             pyhd8ed1ab_0    conda-forge
pyviz_comms               2.3.2                      py_0    pyviz
pyyaml                    6.0             py310h90acd4f_5    conda-forge
pyzmq                     25.1.0          py310h998be00_0    conda-forge
qt-main                   5.15.8              heb9da5c_15    conda-forge
qtconsole                 5.4.3              pyhd8ed1ab_0    conda-forge
qtconsole-base            5.4.3              pyha770c72_0    conda-forge
qtpy                      2.3.1              pyhd8ed1ab_0    conda-forge
readline                  8.2                  h9e318b2_1    conda-forge
referencing               0.30.0             pyhd8ed1ab_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
rfc3339-validator         0.1.4              pyhd8ed1ab_0    conda-forge
rfc3986-validator         0.1.1              pyh9f0ad1d_0    conda-forge
rich                      13.5.0                   pypi_0    pypi
rpds-py                   0.9.2           py310h3461e44_0    conda-forge
rtree                     1.0.1           py310had9ce37_2    conda-forge
scikit-learn              1.3.0           py310hd2c063c_0    conda-forge
scipy                     1.10.1          py310h3900cf1_3    conda-forge
send2trash                1.8.2              pyhd1c38e8_0    conda-forge
setuptools                68.0.0             pyhd8ed1ab_0    conda-forge
shapely                   2.0.1           py310h3ddfc5e_1    conda-forge
sip                       6.7.10          py310h9e9d8ca_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
smmap                     5.0.0                    pypi_0    pypi
snappy                    1.1.10               h225ccf5_0    conda-forge
sniffio                   1.3.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
sqlite                    3.42.0               h2b0dec6_0    conda-forge
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
streamlit                 1.22.0                   pypi_0    pypi
tenacity                  8.2.2              pyhd8ed1ab_0    conda-forge
terminado                 0.17.1             pyhd1c38e8_0    conda-forge
threadpoolctl             3.2.0              pyha21a80b_0    conda-forge
tiledb                    2.13.2               h8b9cbf0_0    conda-forge
tinycss2                  1.2.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h5dbffcc_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tornado                   6.3.2           py310h6729b98_0    conda-forge
tqdm                      4.65.0             pyhd8ed1ab_1    conda-forge
traitlets                 5.9.0              pyhd8ed1ab_0    conda-forge
typing-extensions         4.7.1                hd8ed1ab_0    conda-forge
typing_extensions         4.7.1              pyha770c72_0    conda-forge
typing_utils              0.1.0              pyhd8ed1ab_0    conda-forge
tzcode                    2023c                hb7f2c08_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
tzlocal                   5.0.1                    pypi_0    pypi
unicodedata2              15.0.0          py310h90acd4f_0    conda-forge
uri-template              1.3.0              pyhd8ed1ab_0    conda-forge
urllib3                   2.0.4              pyhd8ed1ab_0    conda-forge
validators                0.14.0                     py_0    bioconda
wcwidth                   0.2.6              pyhd8ed1ab_0    conda-forge
webcolors                 1.13               pyhd8ed1ab_0    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          1.6.1              pyhd8ed1ab_0    conda-forge
wheel                     0.41.0             pyhd8ed1ab_0    conda-forge
widgetsnbextension        4.0.8              pyhd8ed1ab_0    conda-forge
xarray                    2023.4.2           pyhd8ed1ab_0    conda-forge
xerces-c                  3.2.4                h90c7484_2    conda-forge
xorg-libxau               1.0.11               h0dc2134_0    conda-forge
xorg-libxdmcp             1.1.3                h35c211d_0    conda-forge
xyzservices               2023.7.0           pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h775f41a_0    conda-forge
yaml                      0.2.5                h0d85af4_2    conda-forge
zeromq                    4.3.4                he49afe7_1    conda-forge
zipp                      3.16.2             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               h8a1eda9_5    conda-forge
zlib-ng                   2.0.7                hb7f2c08_0    conda-forge
zstd                      1.5.2                h829000d_7    conda-forge
```

**Comment by ekatef** [comment]
Fixed by #4

**Comment by ekatef** [comment]
Fixed by #4 🎉
